### DISCUSSION TITLE
Mark a delayed job as completed or failed

### DISCUSSION THREAD

**Discussion by minhduc17801** [taskforcesh/bullmq#2703] [Q&A]
I have a queue that will mainly process jobs with delay. Sometimes jobs will no longer need to be processed but I don't want to remove it but rather move it to completed or failed (preferably with a reason).

**Comment by manast** [comment]
One way could be to promote the delayed job and then do a noop in the processor https://api.docs.bullmq.io/classes/v5.Job.html#promote
